src/neptune_scale/util/logger.py

```diff
@@ -13,7 +13,7 @@
 class NeptuneWarning(Warning): ...
 
 
-LOG_FORMAT = "{blue}%(name)s{end}:{bold}%(levelname)s{end}: %(message)s"
+LOG_FORMAT = "%(asctime)s {blue}%(name)s{end}:{bold}%(levelname)s{end}: %(message)s"
```

Comment:
A drive-by change, but hopefully ok - not having this is driving me crazy when testing.